resources/lang/de/devices.php

```diff
@@ -59,4 +59,9 @@
   'device_info' => 'Device info',
   'repair_info' => 'Repair info',
   'event_info' => 'Event info',
+  'weight' => 'Weight',
```

Comment:
In general we don't need to duplicate the keys across lang files for other languages, they will take by default from the en files.  (But it doesn't hurt)

Reply:
As discussed I'll just add them into en.